README.md

```diff
@@ -8,19 +13,7 @@
 - 🔥 **Burner Wallet & Local Faucet**: Quickly test your application with a burner wallet and local faucet.
 - 🔐 **Integration with Wallet Providers**: Connect to different wallet providers and interact with the Ethereum network.
 
-## Contents
-
-- [Requirements](#requirements)
-- [Quickstart](#quickstart)
-- [Deploying your Smart Contracts to a Live Network](#deploying-your-smart-contracts-to-a-live-network)
-- [Deploying your NextJS App](#deploying-your-nextjs-app)
-  - [Scaffold App Configuration](#scaffold-app-configuration)
-- [Interacting with your Smart Contracts: SE-2 Custom Hooks](#interacting-with-your-smart-contracts-se-2-custom-hooks)
-- [Disabling Type & Linting Error Checks](#disabling-type-and-linting-error-checks)
-  - [Disabling commit checks](#disabling-commit-checks)
-  - [Deploying to Vercel without any checks](#deploying-to-vercel-without-any-checks)
-  - [Disabling Github Workflow](#disabling-github-workflow)
-- [Contributing to Scaffold-ETH 2](#contributing-to-scaffold-eth-2)
+![Debug Contracts tab](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/aa48ebcb-62f3-46d6-96e0-4931d610442e)
```

Comment:
Hmm, why the image doesn't match the real app? It can confuse newcomers.

Btw it looks awesome, I'd be glad to help update the UI if we have designs for it

Reply:
> Hmm, why the image doesn't match the real app? It can confuse newcomers.
> 
> Btw it looks awesome, I'd be glad to help update the UI if we have designs for it

The origin from the asset was for a Poster (of ETHDenver I believe), and then adapted for the website. Designer decided to adapt real assets to make them fit better for the website. Not sure if that UI from Debug Contracts asset wants to be implemented to SE-2.

Thinking about the "fake asset":
- For the website it makes sense to me since the objective is to look it more clear in the reduced space we got for it.
- For the Documentation, we currently don't have any, but when we add them I agree they should be real screenshots, and as updated as posible, since the objective is to guide our users throught their questions.
- For the repo README, I'm not sure tbh. I think you might be right and real screenshot may fit better, but matching the website asset may make sense also. 

Would love to hear more thoughts before updating them

Reply:
This looks great, tysm @Pabl0cks!!! 

For the README I agree that the real screenshot would be a better fit. Also agree with your take for the website and the documentation too.

Reply:
Yup, agree for Readme and Docs(in case in the future) let's have actual SS. For the website, the designed one fits well and looks good with the overall theme 🙌

Reply:
Thx @rin-st @technophile-04 and @edakturk14 for the feedback! 

I've just pushed the update of the SE-2 screenshot asset. 
I took the designed sshot size and style, and updated the content with the real content for a ERC20 contract. Tweaked the screenshot a bit to mantain the bigger menu item size for better readability, but if you don't like it I can just set everything 100% real.

If you like the idea but would prefer better execution, I can ask Andrea to do it :)

Thanks again ♥🙌

Reply:
Thanks @Pabl0cks ! Looks great to me